Makefile.doc

```diff
@@ -67,9 +67,8 @@ sphinx-%: $(SPHINX_DEPS)
 sphinx-pdf: sphinx-latex
 	+$(MAKE) -C $(SPHINXBUILDDIR)/latex
 
-sphinx:
-	+$(MAKE) sphinx-html
-	+$(MAKE) sphinx-pdf
+sphinx:\
+  sphinx-html sphinx-pdf
```

Comment:
Are you sure about this? I don't think it's safe to run sphinx-html and sphinx-pdf in parallel.